hack/cluster-deploy.sh

```diff
@@ -56,6 +56,17 @@ if [[ "$KUBEVIRT_PROVIDER" =~ os-* ]] || [[ "$KUBEVIRT_PROVIDER" =~ okd-* ]]; th
     _kubectl adm policy add-scc-to-user privileged admin
 fi
 
+if [[ "$KUBEVIRT_PROVIDER" =~ .*sriov.* ]]; then
+    #enable feature gate
+    _kubectl patch configmap kubevirt-config -n kubevirt --patch "data: 
+  feature-gates: DataVolumes, CPUManager, LiveMigration, SRIOV"
```

Comment:
can we read the old value and just add SRIOV? It's already out of sync with `manifests/testing/kubevirt-config.yaml.in` now